fix(whatsapp): content templates

cullenwatson opened the pull request:
* fetches their content templates via serverless
* allows user to select template from dropdown
* fixes conversation not opening if conversation already exists
<img width="289" alt="image" src="https://github.com/user-attachments/assets/5a45cd95-85f4-448f-bf2f-35e3ce552b55">

cullenwatson committed:
fix(whatsapp): content templates

dahamilton-twilio reviewed (commented):
Thanks for you contribution @cullenwatson.

Happy to take a look next week and test it out. I am assuming your templates don't include variables? I would like to try and find the time to solve for that at some point but this is a great first step given the upcoming changes.

dahamilton-twilio commented on src/components/OutboundMessagePanel/OutboundMessagePanel.Container.js:
```diff
@@ -12,16 +12,13 @@ import {
   MessageContainer,
   SendMessageContainer,
   MessageTypeContainer,
-  OfflineContainer,
```
Did the correct version get checked in or am I missing something? Aren't these required? thanks

asugai replied:
I just deployed this plugin using the updates that @cullenwatson made on this PR - those import do need to be added back in @dahamilton-twilio

dahamilton-twilio replied:
Thanks @asugai - appreciate the collaboration. It is on my todo list to get this into into main. Did you make any other changes for your use case that would be useful?

cullenwatson commented:
Ah yea my templates didn't need variables. Prob needed to make this pr complete

dahamilton-twilio commented:
> Ah yea my templates didn't need variables. Prob needed to make this pr complete

All good. Step in the right direction in advance of changes later in the year. Thanks

dahamilton-twilio committed:
Updated PR

+ secure the getContentTemplates endpoint
+ make Content Templates optional - (making this feature default enabled will be done when support for variables is added)
+ added private asset file that allows the content templates that are returned to be configurable
